src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -181,6 +182,13 @@ void noAnimatedStickerConversionNeeded() throws Exception {
 		assertThat(result, is(nullValue()));
 	}
 
+	@Test
+	void isAnimationLowFpsCompliant() throws Exception{
```

Comment:
I'd give it a name resembling the other similar test since they are related
```suggestion
	void noLowFpsAnimatedStickerConversionNeeded() throws Exception{
```

Reply:
Updated isAnimationLowFpsCompliant() with noLowFpsAnimatedStickerConversionNeeded()